doc/upgrade-cluster.md

```diff
@@ -118,15 +111,12 @@ The upgrade workflow is as follows:
   - If any of the component pods has errors, the upgrade process will stop (`Upgrade` condition to `False`) and operator will set the `Error` condition to `True` and indicate the reason.
 - When all components are upgraded, the `Upgrade` condition will be set to `False` and `status.currentVersion` will be updated to the desired version.
 
+### Pravega Segment Store upgrade
 
-### BookKeeper upgrade
-
-BookKeeper is the first component to be upgraded as recommended in [BookKeeper documentation](https://bookkeeper.apache.org/docs/4.7.3/admin/upgrade/).
-
-In Kubernetes, BookKeeper is deployed as a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) due to its requirements on:
+Pravega Segment Store is the first component to be upgraded. The Segment Store is deployed as a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) due to its requirements on:
 
-- Persistent storage: each bookie has three persistent volume for ledgers, journals, and indices. If a pod is migrated or recreated (e.g. when it's upgraded), the data in those volumes will remain untouched.
-- Stable network names: the `StatefulSet` provides pods with a predictable name and a [Headless service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) creates DNS records for pods to be reachable by clients. If a pod is recreated or migrated to a different node, clients will continue to be able to reach the pod despite changing its IP address.
+- Persistent storage: each segmentstore need access to a persistent volume to store the cache. Losing the data on the cache would not be critical because the Segment Store would be able to recover based on the data in BookKeeper, but it would have a performance impact that we want to avoid when migrating, recreating, or upgrading Segment Store pods.
```

Comment:
Please remove this section on Persistent Storage as it is not relevant any more. Starting version 0.7, there is no "persistent volume" cache in segment store.